### PR TITLE
Weather App by Agnes

### PR DESCRIPTION
**Pull request by Agnes94**
Here's the link to the site: https://tender-lovelace-d6ec5f.netlify.com/

### CODE REVIEW

File: Code/script.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+)
+  //The Json from the API//
+
+  .then(response => {
+    return response.json();
+  })
+
+  .then(json => {
+    const theCity = document.getElementById("city");
+    const theTemp = document.getElementById("temp");
```
Comment (ericaglimsholt) [inline Code/script.js]: What does temp stands for? Temperature? Temporary? Maybe you can make this name more describing

**Comment by Agnes94** [comment]
I actually don’t understand why is that when I open the site using mobile the sunrise and sunset times are different When I check it in chrome using the developer tools for Iphone X it only shows the hours and the minutes, but if I check the site from my phone it looks like this: 7:10:49 AM for instance. 

I struggled with the 5 day forecast part so I basically copied Damien’s code and I also found it difficult to style that part, so I just kept it very simple.

File: Code/script.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    theWeather.innerHTML = `${json.weather[0].description}`;
+    theSunrise.innerHTML = `<img src="Assets/noun_sunrise_1550440.png" alt="icon" width=60px></br>${sunriseTime}`;
+    theSunset.innerHTML = `<img src="Assets/noun_sunset_1632877.png" alt="icon" width=60px></br>${sunsetTime}`;
+
+    //Conditions for changing icons next to weather description//
+
+    if (json.weather[0].description === "broken clouds") {
+      theWeather.innerHTML = `<img src="Assets/broken-clouds.png" alt="icon" width=80px">`;
+    } else if (json.weather[0].description === "clouds") {
+      theWeather.innerHTML = `<img src="Assets/scattered-clouds" alt="icon" width=\"70px\">`;
```
[the remark this answers is not in the document]
Comment (Agnes94) [reply Code/script.js]: Simply by accident!

**Comment by ericaglimsholt** [comment]
> I actually don’t understand why is that when I open the site using mobile the sunrise and sunset times are different When I check it in chrome using the developer tools for Iphone X it only shows the hours and the minutes, but if I check the site from my phone it looks like this: 7:10:49 AM for instance.
> 
> I struggled with the 5 day forecast part so I basically copied Damien’s code and I also found it difficult to style that part, so I just kept it very simple.

It appears different because you are using `toLocaleTimeString()`. So what that does is to look what the local time depending where you are/what's in your settings. So your phone thinks that you are in a different zone than Sweden. That happens to me as well, because I have my phone in english (and I think the zone is US per default on iPhones) :)

**Pull request closed without merging**